TextToSpeech/TextToSpeechImplementation.h

```diff
@@ -86,6 +86,26 @@ namespace Plugin {
             const Event _event;
             const JsonValue _params;
         };
+        private:
+            struct InputValidation
+            {
+                InputValidation &operator=(const InputValidation &copy) = delete;
+                InputValidation(const InputValidation &copy) = delete;
+                InputValidation &operator=(InputValidation &&copy) = delete;
+                InputValidation(InputValidation &&copy) = delete;
+                ~InputValidation() = default;
+
+                static ValidationManager &Instance()
+                {
+                    static InputValidation s_instance;
+                    return s_instance.m_validationManager;
+                }
+                private:
+                InputValidation() = default;
+
+                private:
+                ValidationManager m_validationManager;
+            };
```

Comment:
Why is there again a InputValidation seen here, I saw one in JsonRpc implementation!!!